contracts/test/setup.ts

```diff
@@ -0,0 +1,91 @@
+import * as fs from 'fs'
```

Comment:
What's the benefit of keeping hardhat around?
At this point, we use it just for integration tests, right?

Reply:
^ Some functionality is hard or impossible to reproduce in cairo test. For example, the replace class syscall isn't supported by cairo test. Also, the test harness setup logic for aggregator is not easily achievable in cairo-test because it involves a lot of cryptography.